samples/dfu/fw_loader/ble_mcumgr/sample.yaml

```diff
@@ -0,0 +1,13 @@
+sample:
+  description: Minimal Firmware Loader sample
```

Comment:
Please consider using the same name as in README.md file: `Minimal Bluetooth LE SMP firmware loader`